File: source/code/php/authentication/create_account_linkedin_providerdata_access_token.php
$providerRequest = new LinkedInProviderRequest(array(
                        "accessToken" =>"y29.1.AADN_Xo2hxQflWwsgCSK-WjSw1mNfZiv4"
                    ));
$result = $application->getAccount(request);
$account = $result->getAccount();

File: source/code/java/authentication/create_account_linkedin_providerdata_access_token.java
import com.stormpath.sdk.provider;
import com.stormpath.sdk.account;

ProviderAccountRequest request = Providers.LINKEDIN.account()
            .setAccessToken(accessToken)
            .build();

ProviderAccountResult result = application.getAccount(request);
Account account = result.getAccount();
